docs/architecture/cloud-native/application-resiliency-patterns.md

```diff
@@ -31,7 +31,7 @@ Next, let's expand on retry and circuit breaker patterns.
 
 In a distributed cloud-native environment, calls to services and cloud resources can fail because of transient (short-lived) failures, which typically correct themselves after a brief period of time. Implementing a retry strategy helps a cloud-native service handle these scenarios.
 
-The [Retry pattern](https://docs.microsoft.com/azure/architecture/patterns/retry) enables a service to retry a failed request operation a (configurable) number of times with an exponentially increasing wait time. Figure 6-4 shows a retry in action.
+The [Retry pattern](/azure/architecture/patterns/retry) enables a service to retry a failed request operation a (configurable) number of times with an exponentially increasing wait time. Figure 6-4 shows a retry in action.
```

Comment:
The full link is better for folks seeing the article on GitHub. We usually use the relative path for docsets that have VS offline books, because that's the only reason we use these in the first place. Azure doesn't have a VS offline book, so I'd keep those as-is.  This applies to other changes in this PR.